Automattic/woocommerce-payments
# Implement session cookie for WooPay checkout flow

#### Changes proposed in this Pull Request

This PR enhances user experience during the checkout process by introducing a session cookie mechanism. Currently, when users are in the WooPay checkout and navigate back to the store checkout they may end up on WooPay again unintendedly depending on how they navigate through the store. This behavior always happens particularly in the WooPay Direct Checkout flow.

To mitigate this behavior, I have implemented a session cookie feature that allows the user to bypass the WooPay checkout flow (during that specific user session only) if they choose to go back during the process.









#### Testing instructions

See testing instructions in https://github.com/Automattic/woopay/pull/2630.






-------------------


**Post merge**
